src/collection/query/find.ts

```
@@ -12,12 +12,14 @@ import type {
   InferSchemaOutput,
 } from "../../schema/type-helpers";
 import type { Pretty, TrueKeys } from "../../type-helpers";
+import { PipelineStage } from "../types/pipeline-stage";
```

Biome

[error] All these imports are only used as types.

Importing the types with import type ensures that they are removed by the compilers and avoids loading unnecessary modules.
Safe fix: Use import type.


(lint/style/useImportType)